index.bs

```diff
@@ -2999,12 +2998,11 @@ When verifying a given {{PublicKeyCredential}} structure (|credential|) and an {
 
 1. Verify that the <code>[=rpIdHash=]</code> in |aData| is the SHA-256 hash of the [=RP ID=] expected by the [=[RP]=].
 
+1. Verify that the [=User Present=] bit of the <code>[=flags=]</code> in |authData| is set.
```

Comment:
I think the proposed changes per se are good, but here in [7.2. Verifying an authentication assertion](https://w3c.github.io/webauthn/#verifying-assertion) we seem to have an issue with using both |aData| and |authData| interchangeably, yes? 

I _think_ all instances of |aData| and |authData| in this section ought to be changed to be either one or the other. If I am not correct, then I'm thinking we ought to have a Note explaining the difference between them. 

Otherwise this seems fine.

Reply:
Thanks, good catch. This looks like two unrelated copy-and paste errors (this PR and #811). I agree we'd be better off using the same variable name in both algorithms, I suggest we choose `|authData|`.

Reply:
using `|authData|` is fine by me.

Reply:
I'll change to `|aData|` in this PR and submit a separate PR to change from `|aData|` and `|adata|` to `|authData|`.

Reply:
Opened issue #879.

Reply:
OK, thanks. I note the new PR to "change from `|aData|` and `|adata|` to `|authData|`" has not yet appeared but we now have issue #879 for that, thus this plan sounds fine so approving this PR (and plan). thx!